index.html

```diff
@@ -81,7 +111,7 @@ <h1 class="Text__intro"> <strong>Hey there. I’m Solomon </strong></h1>  <br> <
                     </a>
             </div>
             <div class="recent__work">
-                <span class="recent__work--text">My Recent <br> Works</span>
+                <span class="recent__work--text">My Recent Works</span>
```

Comment:
- [ ] Kindly note that thus us supposed to use a heading tag `<h2>` to `<h6>` this is so that it meets the sematic requirememnts.
- [ ] Also kindly note that your design currently has some missing paths to images to this section as well as he about section. Kindly fix the path so that your project can  be complete
<img width="1427" alt="Screenshot 2022-09-30 at 14 07 15" src="https://user-images.githubusercontent.com/48546992/193259590-506f29a2-6eac-403f-aafc-234562bec516.png">

<img width="1427" alt="Screenshot 2022-09-30 at 14 07 05" src="https://user-images.githubusercontent.com/48546992/193259727-195fcc67-1bad-4e0f-8ab6-9fcd556d0f5b.png">

Reply:
Hello @Nyame-Wolf Thank you for your review; However this this image is actually loading where you run the code on a live-server {Vs code extension} . but if you run directly as an html file it would will not be visible.

Reply:
Hello, @Nyame-Wolf are those images are visible on your machine?